sprockets/mixins/metrics/statsd.py

```diff
@@ -191,5 +230,14 @@ def install(application, **kwargs):
     if 'port' not in kwargs:
         kwargs['port'] = os.environ.get('STATSD_PORT', '8125')
 
+    if 'proto' not in kwargs:
+        if "STATSD_PROTO" in os.environ:
```

Comment:
No need to check if it's in the environment - the result will always be a string if it is else `None` or the default. You can reduce this to a single line.

```
kwargs['proto'] = os.environ.get('STATSD_PROTO', 'udp')